src/test/unit/pid_unittest.cc

```diff
@@ -143,6 +143,7 @@ void setDefaultTestSettings(void) {
     pidProfile->abs_control_gain = 0,
     pidProfile->launchControlMode = LAUNCH_CONTROL_MODE_NORMAL,
     pidProfile->launchControlGain = 40,
+    pidProfile->race_mode = false,
```

Comment:
Same point as above on the settings. "race_mode" is too generic a name for the element. It should give a clear indication it's related to angle mode.

Reply:
Done - renamed